american-journal-of-archaeology.csl

```diff
@@ -356,14 +371,13 @@
     <layout suffix=".">
       <text macro="contributors" suffix=". "/>
       <text macro="date" suffix=". "/>
-      <text macro="day-month" text-case="capitalize-first" suffix=". "/>
       <text macro="title"/>
       <text macro="description"/>
-      <text macro="secondary-contributors" prefix=". "/>
+      <text macro="secondary-contributors" prefix=" "/>
       <text macro="container-title"/>
       <text macro="container-contributors" prefix=", "/>
       <text macro="locators-chapter"/>
-      <text macro="edition" prefix=". "/>
+      <text macro="edition" prefix=". " strip-periods="true"/>
```

Comment:
The strip-periods here is a bit odd, especially when you think of non-numeric entries like "2nd rev. and ext. edition" -- what periods are you trying to remove?

Reply:
Because the locators macro that follows it uses a period as a prefix to books, so you can get two periods in a row if the edition ends with one, as in your example. 

IF there's a better way to handle that, great.